yb_migrate/cmd/exportData.go

```diff
@@ -54,6 +54,7 @@ func init() {
 
 func exportData() {
 	fmt.Printf("export of data for source type as '%s'\n", source.DBType)
+	log.Infof(fmt.Sprintf("export of data for source type as '%s'\n", source.DBType))
```

Comment:
No need of the `fmt.Sprintf()`. `log.Infof()` accepts format string as first argument. Whenever you see a trailing `f` in the name e.g. `Sprintf`, `Infof`, `Warnf`, `Fatalf` it indicates that the function accepts format string.